OrbitCore/ConnectionManager.cpp

```diff
@@ -25,6 +26,7 @@
 #include <streambuf>
 #include <sstream>
 #include <iostream>
+#include <cereal/types/vector.hpp>
```

Comment:
We should include "Serialization.h" in case we change the implementation at some point.